jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/form/FormsSizingTests.java

```diff
@@ -86,17 +87,9 @@ public void isValidationTest() {
         formsSizing.smallSelect.is()
                 .displayed()
                 .selected("Small select");
-        formsSizing.largeTextField
-                .placeholder()
-                .equalsIgnoreCase(placeholderLarge);
-        formsSizing.defaultTextField.is()
-                .enabled()
-                .text(is(placeholderDefault));
-//                .placeholder(placeholderDefault);
-        formsSizing.smallTextField.is()
-                .enabled()
-                .text(is(placeholderSmall));
-//                .placeholder(placeholderSmall);
+        assertTrue(formsSizing.largeTextField.placeholder().equalsIgnoreCase(placeholderLarge));
+        assertTrue(formsSizing.defaultTextField.placeholder().equalsIgnoreCase(placeholderDefault));
+        assertTrue(formsSizing.smallTextField.placeholder().equalsIgnoreCase(placeholderSmall));
```

Comment:
don't miss is().enabled() checks and you should use the same jdi checks as:
```
        formsSizing.defaultTextField.is()
                .enabled()
                .text(is(placeholderDefault));
```
instead of testng